### ISSUE TITLE
KCD 2020 Munich, Germany

### ISSUE THREAD

**Issue by mkorbi** [cncf/kubernetes-community-days#182]
### Event City: 
Munich/München, Bavaria, Germany
Second or third July week 2020

### Submission checklist:
* [X] All organizers signed off to the event guidelines and CNCF Code of Conduct.
* [X] Method to accept payments including sponsorships and payout expenses
* [X] One organizer is a CNCF Member employee, Ambassador, or CNCF Project Maintainer
* [X] Minimum of 3 organizers from 3 different organizations
* [X] Each organizer needs to live in or just outside of the KCD location
* [X] No other Kubernetes Community Days event in the same city
* [X] Agree to use Eventbrite for registration
* [X] Agree to use Sched for organizing the program page

### Organizer 1: 
Max Körbächer, Storm Reply, Senior Cloud Solution Architect, m.koerbaecher@reply.de
Gerald Zabos, Storm Reply, Senior Consultant, g.zabos@reply.de
### Organizer 2: 
Alexander Knippig, noris network, IT Systems Engineer, alexander.knipping@noris.de
### Organizer 3: 
Christina Kari, Loodse, Senior Account Executive, christina@loodse.com
### Organizer 4: 
Tom Wiesseckel, Mayflower, Caretaker, tom.wiesseckel@mayflower.de

Storm, Loodse and noris have additional personal support in the background. Storm Reply and Loodse are experienced in organizing conferences and big events.

### List of Sponsors:
* Storm Reply
* Loodse
* noris network
* Mayflower

**Comment by christinevblum** [comment]
Please update the issue as you complete the items on the checklist. Once everything is complete we'll sign off on your team creating the event site.

**Comment by ThorstenHans** [comment]
Hey @mkorbi,

If you’re looking for an additional organizer, let me know. 

-Thorsten

**Comment by mkorbi** [comment]
@ThorstenHans pn you at k8s slack

**Comment by mkorbi** [comment]
@christinevblum all done and updated :)

**Comment by dankohn** [comment]
Great! Could you please specify how you're planning on accepting and making payments? Will Storm Reply or Loodse act as the fiscal sponsor?

**Comment by mkorbi** [comment]
Storm will act as a fiscal sponsor. We will do out through our corporate bank account, where we also can create from invoices for sponsors.

**Comment by KTan1226** [comment]
Hi @mkorbi- We need you to list 3 companies that have agreed to sponsor and then we can approve.

**Comment by mkorbi** [comment]
@KTan1226 done :)

**Comment by KTan1226** [comment]
Great news, @mkorbi. You are approved to create your site.

**Comment by KTan1226** [comment]
@mkorbi Can you please share the Slack username for each organizer? I'll use this to add the team to the KCD organizers Slack channel. Thanks!

**Comment by mkorbi** [comment]
@KTan1226 for sure the users are
mkoerbi 
gmzabos 
sissi 
obitech 
divb

**Comment by KTan1226** [comment]
Hi @mkorbi- can you share their Slack usernames for CNCF? The KCD channel is under CNCF not the Kubernetes account. Thank you!

**Comment by mkorbi** [comment]
@KTan1226 then the nicks will be
Alex
gmzabos
divb

**Comment by KTan1226** [comment]
Thanks @mkorbi!

**Comment by dankohn** [comment]
Ready to create your site? Can we help?

**Comment by mkorbi** [comment]
Yes, we are ready, forked and working on it

**Comment by mkorbi** [comment]
When we will get the Sched account?

**Comment by obitech** [comment]
@dankohn I've just created the PR: #307

**Comment by KTan1226** [comment]
Hi @mkorbi. To answer your question on Sched, we will grant access once your proposals are selected and your agenda is confirmed.

**Comment by obitech** [comment]
@KTan1226 @dankohn given the current situation and uncertainty how events later this year will be effected, we have decided to postpone KCD Munich 2020 to an unspecified date. 

What would be the best way forward now? Maybe we can update the website with a notice, leave it up for a couple of weeks and then pull the event from the site until a new date has been agreed upon?

**Comment by KTan1226** [comment]
Hi @obitech. Yes, please update the website with a note for now and then remove the page in a few weeks if needed. If you would like guidance on what to include in your note, we recommend referencing the following: https://events.linuxfoundation.org/2020/03/13/novel-coronavirus-update/

Let me know if you have any questions or need help. Thanks!

**Comment by xmulligan** [comment]
@mkorbi We are relaunching the KCD program, including virtual events. We would love to have one in Munich if you are still interested! You can read [the blog post here](https://www.cncf.io/blog/2021/03/01/relaunching-kubernetes-community-days-with-kcd-africa-bengaluru/). If you have any questions, please contact me at kcd@cncf.io or [book a meeting with me](https://calendly.com/billmulligan/1-1-with-bill-mulligan?month=2021-03).
